packages/babel-parser/test/fixtures/typescript/disallow-jsx-ambiguity/type-parameter-unambiguous-babel-7/input.ts

```diff
@@ -1,2 +1,3 @@
 <T,>() => 1;
 <T,>(x) => 1;
+<T extends unknown>(x) => 1;
```

Comment:
Can you also add
```ts
<T = unknown,>(x) => 1
```

It should be unambiguous because of the trailing comma.

Reply:
Done